src/com/owncloud/android/logs/Log.java

```diff
@@ -0,0 +1,51 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   @author David González Verdugo
+ *   Copyright (C) 2018 ownCloud GmbH.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License version 2,
+ *   as published by the Free Software Foundation.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.owncloud.android.logs;
```

Comment:
Do you think it necessary to add a new package just for a single class? Couldn't we put it eg. into `utils`?

Reply:
I didn't find a proper package to put it. This new class is not a `utils` class, is kind of a model class but it would't fit in `datamodel` package either since that package contains things to save in our local database and is not the case.

Reply:
Hm and just keeping it in the root folder then? Tho I know this is not our style.

Reply:
I would rather keep it in the logs package, in the root folder we don't have model classes.